features/past_events_page.feature

```diff
@@ -0,0 +1,20 @@
+Feature: Use the past events page
+
+  As visitor to the website
+  So that I can browse previous events
+  I want to be able to see the past events and their details
+
+  Background:
```

Comment:
this test will only work until 4/20, which is not a good day for anything to break unexpectedly. Try to find a way that the test is a little more permanent.